### PR TITLE
labs commands

### PR DESCRIPTION
**Pull request by dickeyxxx**

### CODE REVIEW

File: commands/labs/index.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    line = `${line}  ${f.description}`;
+    cli.log(line);
+  }
+}
+
+function* run (context, heroku) {
+  let features = yield {
+    currentUser: heroku.get('/account'),
+    user: heroku.get('/account/features'),
+    app: context.app ? heroku.get(`/apps/${context.app}/features`).catch(() => null) : null,
```
Comment (ransombriggs) [inline commands/labs/index.js]: @dickeyxxx Why do you have the catch here?

Comment (dickeyxxx) [inline commands/labs/index.js]: not sure, I'll take it out

**Comment by dickeyxxx** [comment]
ready for review again @ransombriggs

**Comment by ransombriggs** [comment]
@dickeyxxx LGTM

**Pull request merged**